old-ui/app/components/connect-hardware/account-list.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ethNetProps from 'eth-net-props'
+import { default as Select } from 'react-select'
+import Button from '../../../../ui/app/components/button'
+
+class AccountList extends Component {
+    constructor (props, context) {
+        super(props)
+    }
+
+    getHdPaths = () => {
+      return [
+        {
+          label: `Ledger Live`,
+          value: `m/44'/60'/0'/0/0`,
+        },
+        {
+          label: `Legacy (MEW / MyCrypto)`,
+          value: `m/44'/60'/0'`,
+        },
+      ]
+    }
+
+    goToNextPage = () => {
+      // If we have < 5 accounts, it's restricted by BIP-44
+      if (this.props.accounts.length === 5) {
+        this.props.getPage(this.props.device, 1, this.props.selectedPath)
+      } else {
+        this.props.onAccountRestriction()
+      }
+    }
+
+    goToPreviousPage = () => {
+      this.props.getPage(this.props.device, -1, this.props.selectedPath)
+    }
+
+    renderHdPathSelector = () => {
+      const { onPathChange, selectedPath } = this.props
+
+      const options = this.getHdPaths()
+      return (
+        <div>
+          <h3 className="hw-connect__hdPath__title">Select HD Path</h3>
+          <p className="hw-connect__msg">{`If you don't see your existing Ledger accounts below, try switching paths to "Legacy (MEW / MyCrypto)"`}</p>
+          <div className="hw-connect__hdPath">
+            <Select
+              className="hw-connect__hdPath__select"
+              name="hd-path-select"
+              clearable={false}
+              value={selectedPath}
+              options={options}
+              onChange={(opt) => {
+                onPathChange(opt.value)
+              }}
+            />
+          </div>
+        </div>
+      )
+    }
+
+    capitalizeDevice = (device) => {
```

Comment:
This function seems to do the same as `capitalizeFirstLetter`.

Reply:
Thanks. Duplicate has been removed https://github.com/poanetwork/nifty-wallet/pull/183/commits/256e58527485a7622f8049d2c03f53aca058b994